dev-lang/vala/vala-0.36.12.recipe

```diff
@@ -75,22 +100,16 @@ INSTALL()
 {
 	make install
 
-	# remove libtool library files
-	rm $libDir/lib*.la
+	rm -f $libDir/*.la
 
 	fixPkgconfig
 
-	packageEntries common \
-		$dataDir/aclocal/vapigen.m4 \
-		$dataDir/aclocal/vala.m4 \
-		$dataDir/vala/Makefile.vapigen
-
-	# Avoid having an empty data/aclocal/ in vala$secondaryArchSuffix
-	rmdir $dataDir/aclocal
+	#devel and common packages
+	packageEntries devel \
+		$developDir
```

Comment:
I've just built vala-0.36.12 on x86_gcc2 (using x86 second arch) and it builds fine.
But I think some files should be moved to the devel package:
```diff
-	#devel and common packages
+	# devel package
 	packageEntries devel \
-		$developDir
+		$developDir \
+		$dataDir/aclocal \
+		$dataDir/vala/Makefile.vapigen
```
It might also be appropriate to move some runtimes, like valac, vapicheck and vapigen, but I'd need to look closer.